Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -90,6 +90,19 @@ def run
           end
         end
 
+        # Skip formulae that are autobumped by BrewTestBot.
+        formulae_and_casks_to_check = formulae_and_casks_to_check.reject do |formula_or_cask|
+          next false if formula_or_cask.respond_to?(:token) # Only formulae are autobumped.
+
+          autobump_file = formula_or_cask.tap.path/".github/autobump.txt"
+          next false unless File.exist?(autobump_file)
+
+          if File.read(autobump_file).include?(formula_or_cask.name)
+            odebug "Skipping #{formula_or_cask.name} as it is autobumped."
+            true
+          end
```

Comment:
It would be better if the file could be read outside the loop so that we don't read it over and over again.

Reply:
😳 Thanks for picking this up!